java/client/src/main/java/glide/api/commands/ScriptingAndFunctionsClusterCommands.java

```diff
@@ -13,75 +13,40 @@
 public interface ScriptingAndFunctionsClusterCommands {
 
     /**
-     * Loads a library to Redis unless a library with the same name exists. Use {@link
-     * #functionLoadReplace(String)} to replace existing libraries.<br>
+     * Loads a library to Redis unless a library with the same name exists.<br>
      * The command will be routed to all primary nodes.
      *
      * @since Redis 7.0 and above.
      * @see <a href="https://redis.io/docs/latest/commands/function-load/">redis.io</a> for details.
      * @param libraryCode The source code that implements the library.
+     * @param replace Whether new library overwrites a library with the same name if it exists.
```

Comment:
nit: might be slightly clearer, also do we also want to mention that an error will be thrown if replace is set to false but the library exists? Sorry if I've asked this already, can't remember
```suggestion
     * @param replace Whether the given library should overwrite a library with the same name if it already exists.
```

Reply:
Can you update in all docs?